src/stream/StorageNode.js

```diff
@@ -0,0 +1,9 @@
+export default class StorageNode {
+    constructor(address) {
+        this._address = address
+    }
+
+    getAddress() {
+        return this._address
+    }
+}
```

Comment:
Yeah this seems like it shouldn't be a class, or even an object… it's just a string?